src/components/ec-table/ec-table.vue

```diff
@@ -137,81 +137,81 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import '../../scss/settings/colors/index';
-@import '../../scss/tools/index';
+<style>
+@import '../../styles/tools/scrollbars';
+
+.common-cell-layout {
+  &:last-of-type {
+    padding-right: 16px;
+  }
+}
 
 .ec-table-scroll-container {
-  overflow: auto;
+  @apply tw-overflow-auto;
 
-  @include small-scrollbar;
+  @mixin small-scrollbar;
 }
 
 .ec-table {
+  @apply tw-border-none;
+  @apply tw-pt-16;
+  @apply tw-w-full;
+  @apply tw-relative;
+
   border-spacing: 0;
-  border: none;
-  padding-top: 16px;
-  width: 100%;
-  position: relative; // We need to reset the z-index so the sticky columns and header will not compete with the outer world
-  z-index: 0;
```

Comment:
Any reason to remove the z-index?

Reply:
it's just below, isn't it? (Happened to me too)

Reply:
Then use the tw-z-0 class

Reply:
I think we don't have that class available. I checked and we are using the same css rule in src/components/ec-currency-input/ec-currency-input.vue that is already migrated.

I am happy to add the `tw-z-level-0` class if we agree.

Reply:
Done thanks!